packages/jest-fake-timers/src/legacyFakeTimers.ts

```diff
@@ -429,6 +470,10 @@ export default class FakeTimers<TimerRef> {
     });
   }
 
+  private _fakeRequestAnimationFrame(callback: Callback) {
+    return this._fakeSetTimeout(callback, 1000 / 60);
```

Comment:
it should be called with `now` as an argument as well. https://developer.mozilla.org/en-US/docs/Web/API/window/requestAnimationFrame#parameters

Reply:
Used `this._now`. Ultimately we should use `performance.now()` but that isn't mocked at the moment i.e. use a different clock.

Reply:
continued in https://github.com/facebook/jest/pull/11567#discussion_r650816996